src/bindings/each.js

```diff
@@ -114,15 +115,23 @@ function mustFilterItem(condition, context) {
 /**
  * Extend the scope of the looped template
  * @param   {Object} scope - current template scope
+ * @param   {Object} options - options
  * @param   {string} options.itemName - key to identify the looped item in the new context
  * @param   {string} options.indexName - key to identify the index of the looped item
  * @param   {number} options.index - current index
  * @param   {*} options.item - collection item looped
  * @returns {Object} enhanced scope object
  */
 function extendScope(scope, {itemName, indexName, index, item}) {
-  scope[itemName] = item
-  if (indexName) scope[indexName] = index
+  const options = {
```

Comment:
Is a custom options object necessary here, can't we use the defaults?

Reply:
Yes we can. Everything will work. I don't think there will be any problems without the enumerable option.